src/librustc/ty/mod.rs

```diff
@@ -417,21 +417,6 @@ pub struct AssociatedType<'tcx> {
     pub container: ImplOrTraitItemContainer,
 }
 
-#[derive(Clone, PartialEq, RustcDecodable, RustcEncodable)]
-pub struct ItemVariances {
```

Comment:
Nit: I think I'd have preferred to keep this struct around as a newtype'd Vec, personally. But I don't really care. And I feel guilty even mentioning it. But I will anyway.